vault/core.go

```diff
@@ -477,14 +477,31 @@ func (c *Core) handleRequest(req *logical.Request) (retResp *logical.Response, r
 	// If there is a secret, we must register it with the expiration manager.
 	// We exclude renewal of a lease, since it does not need to be re-registered
 	if resp != nil && resp.Secret != nil && !strings.HasPrefix(req.Path, "sys/renew/") {
+		// Get the SystemView for the mount
+		sysView, err := c.sysViewByPath(req.Path)
+		if err != nil {
```

Comment:
It seems strange that this path could ever result in an error... The log statements should probably be more useful like "[ERR] core: blah blah: err"